tests/contrib/test_zuko.py

```diff
@@ -0,0 +1,56 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import pytest
+import torch
+
+import pyro
+from pyro.contrib.zuko import Zuko2Pyro
+from pyro.infer import SVI, Trace_ELBO
+from pyro.optim import Adam
+
+
+@pytest.mark.parametrize("multivariate", [True, False])
+def test_Zuko2Pyro(multivariate: bool):
```

Comment:
@fritzo anything else worth testing for?

Reply:
I can add a "fake" `rsample_and_log_prob` method to the `dist` object and check that it runs with that.

```python
def rsample_and_log_prob(self, shape):
    x = self.rsample(shape)
    return x, self.log_prob(x)

dist.rsample_and_log_prob = rsample_and_log_prob
```

This would at least test that `ZukoToPyro` works as expected.

Reply:
sounds like a reasonable solution to me

Reply:
It is done!